Source/PangleAdapter.swift

```diff
@@ -58,6 +58,7 @@ final class PangleAdapter: PartnerAdapter {
                 log(.setUpSucceded)
                 completion(nil)
             } else {
+                let error = error ?? NSError(domain: "com.chartboost.mediation.adapter.pangle", code: 0)
```

Comment:
```suggestion
                let error = error ?? self.error(.partnerError)
```
`.partnerError` is already defined as "unknown partner error", so I think this would be better here.

Reply:
+1